stable/atlantis/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.7.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.0.0
+version: 3.0.1
```

```suggestion
version: 3.1.1
```

Another PR was merged today, sorry for the pain.